packages/node/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -293,6 +293,29 @@ describe("AuthCodeRedirectHandler", () => {
       );
     });
 
+    it("cleans up the redirect IRI from the OIDC parameters", async () => {
+      // This function represents the openid-client callback
+      const callback = jest.fn().mockResolvedValueOnce(mockDpopTokens());
+      setupOidcClientMock(undefined, callback);
+      const mockedStorage = mockDefaultRedirectStorage();
+
+      const authCodeRedirectHandler = getAuthCodeRedirectHandler({
+        storageUtility: mockedStorage,
+        sessionInfoManager: mockSessionInfoManager(mockedStorage),
+      });
+
+      await authCodeRedirectHandler.handle(
+        "https://my.app/redirect?code=someCode&state=someState"
+      );
+
+      expect(callback).toHaveBeenCalledWith(
+        "https://my.app/redirect",
+        { code: "someCode", state: "someState" },
+        { code_verifier: "some code verifier", state: "someState" },
```

Comment:
It's not clear from this test where the value "some code verifier" is coming from (i.e., I can see where the values on the prevous line are coming from (i.e., query params on the URI on line 308), but not this value)? Maybe just a comment to explain that the `code_verifier` value is coming from a mock (I assume!)...?